std-bits/table/src/main/java/org/enso/table/data/column/operation/CountNothing.java

```diff
@@ -0,0 +1,64 @@
+package org.enso.table.data.column.operation;
+
+import org.enso.table.data.column.storage.ColumnStorage;
+import org.enso.table.data.column.storage.ColumnStorageWithNothingMap;
+import org.enso.table.data.table.Column;
+import org.graalvm.polyglot.Context;
+
+/** An operation for counting the number of Nothing values in a Column. */
+public class CountNothing {
+  // ** Counts the number of Nothing values in the given column. */
+  public static long apply(Column column) {
+    ColumnStorage storage = column.getStorage();
+    return applyToStorage(storage);
+  }
+
+  // ** Counts the number of Nothing values in the given storage. */
+  public static long applyToStorage(ColumnStorage storage) {
+    if (storage instanceof ColumnStorageWithNothingMap withNothingMap) {
+      return withNothingMap.getIsNothingMap().cardinality();
+    }
+
+    Context context = Context.getCurrent();
+    long count = 0;
+    for (long i = 0; i < storage.getSize(); i++) {
+      if (storage.isNothing(i)) {
+        count += 1;
+      }
+      context.safepoint();
+    }
+    return count;
+  }
+
+  // ** Returns true if all values in the storage are Nothing. */
+  public static boolean anyNothing(ColumnStorage storage) {
+    if (storage instanceof ColumnStorageWithNothingMap withNothingMap) {
+      return withNothingMap.getIsNothingMap().cardinality() > 0;
```

Comment:
```suggestion
      return !withNothingMap.getIsNothingMap().isEmpty();
```

We can do better here and rely on `BitSet.isEmpty` which seems to be O(1) as opposed to O(N) for `cardinality`.

Reply:
> Returns true if this {@code BitSet} contains no bits that are set to {@code true}.